docs/contrib/modifications.md

```diff
@@ -14,6 +19,8 @@ Mode](https://firefox-source-docs.mozilla.org/devtools-user/responsive_design_mo
 or [Chrome/Chromium Device
 Mode](https://developer.chrome.com/docs/devtools/device-mode/) to
 configure your browser with 1080p.
+Screen shots should be added to a directory named `assets` that is a
+sibling of the Markdown file you are adding or editing.
```

Comment:
@colder-is-better: Is there a better way to say this? Is "sibling" too technical here?

Reply:
Yep, I think it is too technical. Maybe try this: "Screen shots should be added to a directory named `assets`, in the same directory as the Markdown file you are adding or editing is".